check_manager/src/check_api.py

```diff
@@ -322,6 +323,18 @@ async def remove_check(
     return await check_backend.remove_check(auth_obj, check_id)
 
 
+@router.post(
+    RUN_CHECK_PATH,
+    status_code=status.HTTP_200_OK,
```

Comment:
Since this never returns any content (apart from error cases), I suggest using 204 return code
```suggestion
    status_code=status.HTTP_204_NO_CONTENT,
```